fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend

```diff
@@ -632,7 +643,8 @@ class PythonUtils {
         «setAttrPropertyHL("delta_t", attr.properties.deltaTime, false)»
         «setAttrPropertyHL("delta_val", attr.properties.deltaValue, false)»
         «setAttrPropertyHL("doc", attr.properties.description.oneLineString, true)»
-        «IF attr.enumLabels!=null»«IF attr.enumLabels.size >0»«setAttrPropertyHL("enum_labels", attr.pythonPipeEnum,  false)»«ENDIF»«ENDIF»«ENDIF»
+«««        «IF attr.enumLabels!=null»«IF attr.enumLabels.size >0»«setAttrPropertyHL("enum_labels", attr.pythonPipeEnum,  false)»«ENDIF»«ENDIF»
+		«ENDIF»
```

Comment:
I don't understand why these two lines are changing.  What does the extra `«««` at the beginning fo the line do?

Reply:
This is commented line which used to generate Enum attribute labels. I have removed it now from the code.